src/router/index.js

```diff
@@ -68,6 +63,11 @@ const routes = [{
   name: 'Register',
   component: () => import('../views/guests/Register.vue'),
 },
+{
+  path: '/',
+  name: 'Login',
```

Comment:
Burada `Login` komponentini import etmek yerine yönlendirme yaparak url'in de değişmesini sağlamalısın.